### DISCUSSION TITLE
Lazer will not run on a fresh windows 10 installation

### DISCUSSION THREAD

**Discussion by Maoman1** [ppy/osu#24849] [Q&A]
### Type

Crash to desktop

### Bug description

I just upgraded this computer a week or two ago and reinstalled windows 10 pro x64 from scratch. I open install.exe and the osu logo shows up in the middle of my screen, a little loading bar underneath it fills up, then... nothing. After 5-10 seconds the process in task manager just disappears. It doesn't even get far enough to generate a log. I tried downloading older releases in case this was a recent thing that broke, even *very* old releases, but nothing works. I'm at a loss as to what to even attempt as far as troubleshooting goes. Stable works, but I want Lazer. 

### Version

2023.914.0

### Logs

As mentioned in the description, it doesn't even make it far enough to create a log.

**Comment by honguyenminh** [comment]
Do you mind give us more info about your configuration, like hardware info, exact Windows version, etc. so we can try to reproduce the issue?

**Comment by Maoman1** [comment]
Is a dxdiag report good enough? Should've thought to include that in the first place.
[DxDiag.txt](https://github.com/ppy/osu/files/12644290/DxDiag.txt)

**Comment by peppy** [comment]
If it's not making a log,`dxdiag` is going to be useless.

Could you please check windows event viewer under "Application" for any crashes there?

**Comment by Maoman1** [comment]
...He asked for hardware info and exact windows version and DxDiag has exactly that, but okay.

Here is what event viewer said, and I copied the report in the "attached file" as well.
[EventViewer.txt](https://github.com/ppy/osu/files/12647503/EventViewer.txt)

**Comment by peppy** [comment]
There should be a second event viewer entry for ".NET exception", can you attach that?

**Comment by Maoman1** [comment]
Closest thing I could find: [net runtime report.txt](https://github.com/ppy/osu/files/12650286/net.runtime.report.txt)

**Comment by peppy** [comment]
No idea sorry. Your game may be corrupt. Try downloading again.

**Comment by Maoman1** [comment]
> The thread is not locked, it's moved to discussions for further discussion. Please follow the link and post there.
> 
> This is standard practice to keep issues limited to those we have confirmed. Sorry if this has upset you.

_Originally posted by @peppy in https://github.com/ppy/osu/issues/24851#issuecomment-1724199253_

I see, I had to refresh the page. All it said was "you cannot comment at this time" so naturally I thought it was locked. I apologize.

Will you please read the rest of my post now?

"Try downloading again"
...I have downloaded install.exe nine separate times, some of them older versions, some of them just repeats of the latest release in case the download was corrupted. None of them work. I already said as much in my first post.

I repeat, this is a fresh install of windows. This isn't an OEM pre-built with all the extras and bloatware pre-installed, this is a custom build with as little extra bullshit as humanly possible because I fucking hate microsoft. It is like two weeks old and literally all I've done so far is browse social media and play Rimworld.

I'm guessing the problem is some dependency I'm missing or it would at the very least try to run, but it doesn't even get that far--so, a basic runtime or library or something. But I'm not gonna just go downloading random shit and hope I stumble onto what I'm missing, I was hoping one of you could tell me what is needed. What basic-ass fundamental something does osu!lazer require that might be missing in a fresh install? If you don't see it on ninite.com, I probably don't have it. Like, I don't think this thing even has a C++ redistributable installed yet. Does it need that? What about Java? I don't fuckin know, it's your code. What is absolutely vital for Lazer to even try to exist on a system?

**Reply by peppy** [reply]
I've confirmed just now it runs fine for me on a fresh WIndows 10 install (we actually have automated testing of this, as well.

If it's a custom build, it's likely missing something which is required to make it run. We can't support this unfortunately. You'll need to figure it out. A starting point may be trying to compile a .NET 6 app yourself and seeing what you run into.

**Reply by bdach** [reply]
I'll also add that dropping a bunch of expletives on us, while omitting the information that this is a "custom build", and giving us only two log excerpts that nothing sensible can be extrapolated out of (one file has nothing interesting in it all, the other has a random nullref that is caused by probably missing whatever else that we can't diagnose out of thin air) is generally not going to be taken very well.

If you're gonna mod windows that much, you should be able to figure out the rest.

**Comment by Maoman1** [comment]
Jesus, sorry for saying nAuGhTy WoRdS, I'll try and remember you've got sensitive little grannies working over there who can't tolerate a little emphasis. 

And I didn't "mod windows" I just said no to all the all the bull"crap" it asks you about when you're installing, and it doesn't have any bloatware. I also provided _exactly_ the information I was asked to provide. If you need more, then just "friggin" tell me so and I'll happily oblige. I literally have no idea what you need, it's not my "gosh darn" code.

If you're gonna respond to a github issue, you should be able to provide something useful to the conversation.

...In any case, I think it's pretty clear now that y'all aren't gonna be any help. Toodles.

**Reply by Coppertine** [reply]
Writing in here just to look through the DxDiag in particlar.. what I find wierd is that your issue not only happens with osu!lazer but also with darktable and gimp  (as well as RimWorld hanging?) (suggesting that this isn't explicitly an osu!lazer issue). I would suggest re-installing your AMD drivers (some dll's seem to be missing that AMD needed), (Possibly a BIOS update?) and to (maybe) install windows 10 WITH the bloatware (it might have extra dlls that osu! requires to run)
